### ISSUE TITLE
Make Share receiver search field accessible

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#11375]
In the "Shares" area, there is a "Search" combo box element that is located in a "div" wrapper.is located in a "div" wrapper. It is not recommended to use the "aria-describedby" attribute together withattribute together with "div" elements, as the output cannot be guaranteed by screen readers.screen readers cannot be guaranteed.

- idea: remove aria-described-by because its not referenced anywhere
